packages/server/package.json

```diff
@@ -81,7 +81,7 @@
   },
   "dependencies": {
     "@cypress/browserify-preprocessor": "1.1.0",
-    "@cypress/commit-info": "^1.2.2",
+    "@cypress/commit-info": "2.0.0",
```

Comment:
v2 returns `null` for values it could not determine, breaking `gift` compatibility, but making it simple to fill values using CI environment variables